.azure-pipelines-templates/cimetrics.yml

```diff
@@ -0,0 +1,30 @@
+jobs:
+- job: CIMetrics
+  displayName: 'CI Metrics'
+  
+  ${{ insert }}: ${{ parameters.env }}
+
+  steps:
+    - checkout: self
```

Comment:
It's mildly annoying to do this just to get metrics.yml, but I don't have a better idea.